.travis.yml

```diff
@@ -15,8 +15,15 @@ install:
   - if [[ $TRAVIS_PYTHON_VERSION != 'pypy'* ]]; then source activate testenv; fi
   - pip install .
   - pip install -r dev-requirements.txt
+  - pip install flake8
```

Comment:
Out of curiosity, could we just put `flake8` in `dev-requirements.txt`?

Reply:
That would be possible but it would push all contributors to install __flake8__ on their local machines which has its pros and cons.  More pros in my book.  ;-)

Reply:
Since it's required to pass on Travis, we should definitely have `flake8` in the dev requirements.